src/com/epam/javacourse/homework7/Main.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.javacourse.homework7;
+
+import java.io.IOException;
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) {
+        String inputFilePath = "src/com/epam/javacourse/resources/poem.txt";
+        String outputFilePath = "src/com/epam/javacourse/resources/poem.txt";
+        ArrayList<String> lines = new ArrayList<>();
+        // Reverse the contents of a file (1)
+        try {
+            FileOperations.reverseFile(inputFilePath, outputFilePath);
+        } catch (IOException e) {
+            System.out.println("An error occurred while reversing the file: " + e.getMessage());
+        }
+
+        //Sort the lines by length (2)
+        try {
+            List<String> sortedLines = StringOperations.sortLinesByLength("src/com/epam/javacourse/resources/poem.txt");
```

Comment:
And that's a duplication, you have a string parameter

Reply:
Fixed